assets/src/dashboard/components/cardGridItem/stories/index.js

```diff
@@ -67,7 +66,7 @@ export const _default = () => {
         <CardTitle
           title="How to be a leader in the apocalpyse"
           author="Rick Grimes"
-          displayDate={moment('04-04-2020').format('MM-DD-YYYY')}
+          displayDate="04-19-2020"
```

Comment:
I know the component just shows the date that's being passed to it as a string, but it still grinds me a bit that it's in the weird (US) order here.